.travis.yml

```diff
@@ -3,6 +3,10 @@ dist: xenial
 language: c
 git:
   submodules: false
+stages:
+  - test
+  - name: deploy
+    if: (NOT type IN (pull_request)) AND (branch = master)
```

Comment:
Aren't pull requests never on master anyways?

Reply:
In the case of a PR, branch equals the base branch (=master).